sonata-project/block-bundle/3.9/config/packages/sonata_block.yaml

```diff
@@ -0,0 +1,2 @@
+sonata_block:
+    default_contexts: [sonata_page_bundle]
```

Comment:
default_context or default_contextS?

Reply:
@OskarStark it is `default_contexts`. See [Configuration.php](https://github.com/sonata-project/SonataBlockBundle/blob/3.x/src/DependencyInjection/Configuration.php#L84). It is required, but if the value is left blank (`default_contexts: []`), there is no errors occured, at least when i use only SonataAdminBundle

May be `default_contexts` should not be required? `sonata_page_bundle` could be appended with recipe for SonataPageBundle!